content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -207,3 +207,69 @@ After a certificate is signed using your preferred method, the certificate and t
 Kubeadm does not support rotation or replacement of CA certificates out of the box.
 
 For more information about manual rotation or replacement of CA, see [manual rotation of CA certificates](/docs/tasks/tls/manual-rotation-of-ca-certificates/).
+
+## Enabling signed kubelet serving certificates {#kubelet-serving-certs}
+
+By default the kubelet serving certificate deployed by kubeadm is self-signed.
+This means a connection from external services like the
+[metrics-server](https://github.com/kubernetes-sigs/metrics-server) to a
+kubelet cannot be secured with TLS.
+
+To configure the kubelets in a new kubeadm cluster to obtain properly signed serving
+certificates you must pass the following minimal configuration to `kubeadm init`:
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+---
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+serverTLSBootstrap: true
+```
+
+If you have already created the cluster you must adapt it by doing the following:
+- Find and edit the `kubelet-config-x.yy` ConfigMap under `kube-system` to include
+the field `serverTLSBootstrap: true` under the `config` key
```

Comment:
We could also write:
```suggestion
 - Find and edit the `kubelet-config-x.yy` ConfigMap in the `kube-system` namespace.
   In that ConfigMap, the `config` key has a
   [kubelet configuration](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) document as
   its value. Edit the kubelet configuration document to set `serverTLSBootstrap: true`.
```

- maybe in a follow-up PR?

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/ recommends changing the ConfigMap that kubelets use (a different name); is that advice related to what's recommended here?

Reply:
> maybe in a follow-up PR?

updated in this PR.

> https://kubernetes.io/docs/tasks/administer-cluster/reconfigure-kubelet/ recommends changing the ConfigMap that kubelets use (a different name); is that advice related to what's recommended here?

kubeadm uses a single ConfigMap, so need to introduce a new ConfigMap with a different name.